docs/Developers/Git_Basics.rst

```diff
@@ -236,7 +236,8 @@ Before you push the commit, ensure that you are not developing on master again
 by running:
 
 ::
-    git status
+
+   $ git status
```

Comment:
2 spaces like everywhere else in the document.

However, this changes the formatting? The `::` actually define the code-style-formatting here^^